apps/api/views.py

```diff
@@ -58,4 +56,7 @@ def update_participant_data(request, participant_id: UUID):
             ParticipantData.objects.create(
                 participant=participant, data=new_data, content_object=experiment, team=experiment.team
             )
-    return HttpResponse()
+    response_body = ""
+    if experiments_not_updated:
+        response_body = {"unsuccessful_updates": experiments_not_updated}
+    return Response(data=response_body)
```

Comment:
I'm open so suggestions here

Reply:
Yeah I was also looking into various others, but none of them "felt right". If we're returning a 422, then we must not update anything in my opinion, otherwise this will be a very interesting side effect. I like your suggestion, it simulates the [207 multi-status approach](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/207)

Reply:
I think fine to exit with an error and not update anything if there are experiment IDs that are incorrect. 

I think it's easier to reason about.

Reply:
Cool cool cool